packer-rhel-ami.json

```diff
@@ -0,0 +1,31 @@
+{
+    "variables": {
+      "ssh_username": "ec2-user",
+      "instance_type": "t2.medium"
+    },
+    "builders": [{
+      "type": "amazon-ebs",
+      "source_ami_filter": {
+        "filters": {
+          "virtualization-type": "hvm",
+          "name": "RHEL-7.7_HVM_GA-20190723-x86_64-1-Hourly2-GP2",
+          "root-device-type": "ebs"
+        },
+        "owners": ["309956199498"],
```

Comment:
@ashish1605 Do we want this value hard-coded or made available via variable?

Reply:
@aabond This is publicly available on AWS. We can leave this value as hard-coded